consensus/polybft/state.go

```diff
@@ -381,19 +379,22 @@ func insertExitEventToBucket(bucket *bolt.Bucket, exitEvent *ExitEvent) error {
 }
 
 // getExitEvent returns exit event with given id, which happened in given epoch and given block number
-func (s *State) getExitEvent(exitEventID, epoch, checkpointBlockNumber uint64) (*ExitEvent, error) {
+func (s *State) getExitEvent(exitEventID, epoch uint64) (*ExitEvent, error) {
 	var exitEvent *ExitEvent
 
 	err := s.db.View(func(tx *bolt.Tx) error {
 		bucket := tx.Bucket(exitEventsBucket)
 
-		key := bytes.Join([][]byte{itob(epoch), itob(exitEventID), itob(checkpointBlockNumber)}, nil)
-		v := bucket.Get(key)
-		if v == nil {
+		key := bytes.Join([][]byte{itob(epoch), itob(exitEventID)}, nil)
```

Comment:
Is there a reason you removed `checkpointBlockNumber` here? Since this function is called on `GenerateExitProof`, where we know the block number, I would leave it as it is, since it was faster before, since it just gets the value by key, without the need to do `Seek`.

Reply:
Yes, there is. We have 2 block numbers there:
* checkpoint block number(when we need to stop create trie)
* exit event block number(that we need there)
In api we are sending only first one.